staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2586,6 +2592,9 @@ type LifecycleHandler struct {
 	// lifecycle hooks will fail in runtime when tcp handler is specified.
 	// +optional
 	TCPSocket *TCPSocketAction `json:"tcpSocket,omitempty" protobuf:"bytes,3,opt,name=tcpSocket"`
+	// Sleep represents the duration in seconds that the container should sleep before being terminated. If the container terminates before the sleep finishes, this action will be interrupted.
```

Comment:
linewrap

Reply:
Can you please add a +featureGate tag here indicating the name of the gate that controls it?